### PR TITLE
Ansible file for RabbitMQ with HA Clustering 

### PR DESCRIPTION
**Pull request by warrenbailey**
**What**
The Ansible file in this pull request will configure two amazon EC2 instances so that they have RabbitMQ installed. It will then cluster them together and setup high availability so the queues are replicated between the two nodes.

**How to test**
1. Follow the instructions in the README to install ansible and provision the EC2 instances
2. Run the python send.py file, this will place a message on a queue called hello
3. Run the python receive.py file, this will read the message from the queue
4. Re-run step 2 and 3, whilst kill the rabbit mq process on each instance.

**Who can review**
Anyone apart from @warrenbailey

### CODE REVIEW

File: ansible/rabbitmq.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+     command: sudo apt-get -qy install rabbitmq-server
+
+  post_tasks:
+   - name: Add users
+     command: sudo rabbitmqctl add_user survey_runner changeit
+
+   - name: Set Permissions
+     command: sudo rabbitmqctl set_permissions survey_runner '.*' '.*' '.*'
+
+#   - name: Create the rabbit env conf file
```
Comment (dhilton) [inline ansible/rabbitmq.yml]: Can these comments be deleted? Do they need to be here?

**Comment by dhilton** [comment]
How do I use the python scripts to verify the install and setup? Is that their purpose?

**Comment by warrenbailey** [comment]
Yes the send.py and receive.py are just there to verify the install.

`python send.py` - will send a message to the hello queue on rabbitmq1
`python receive.py` - will poll the hello queue on rabbitmq2

File: ansible/rabbitmq.yml (hunk 1 above)
Comment (warrenbailey) [inline ansible/rabbitmq.yml]: I'll delete that file, it's no longer used.

**Comment by dhilton** [comment]
Rebased to tidy up some of the documentation - LGTM :+1:

**Pull request merged**